modules/prompt/functions/prompt_sorin_setup

```diff
@@ -32,36 +38,38 @@
 # Load dependencies.
 pmodload 'helper'
 
-prompt_sorin_git_info() {
-  if (( _prompt_sorin_precmd_async_pid > 0 )); then
-    # Append Git status.
-    if [[ -s "$_prompt_sorin_precmd_async_data" ]]; then
-      alias typeset='typeset -g'
-      source "$_prompt_sorin_precmd_async_data"
-      RPROMPT+='${git_info:+${(e)git_info[status]}}'
-      unalias typeset
-    fi
-
-    # Reset PID.
-    _prompt_sorin_precmd_async_pid=0
-
-    # Redisplay prompt.
-    zle && zle reset-prompt
+function _prompt_sorin_async_job {
+  prompt_sorin_style
```

Comment:
Not even necessary, removing this and reverting the style function back into `prompt_sorin_setup`